go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/gballet/go-verkle
 
 go 1.18
 
-require github.com/crate-crypto/go-ipa v0.0.0-20221107233728-e868de2bc50d
+require github.com/crate-crypto/go-ipa v0.0.0-20221110230207-a6614766d44b
 
 require golang.org/x/sys v0.0.0-20220919091848-fb04ddd9f9c8 // indirect
+
+// Temp replace to see the results of https://github.com/crate-crypto/go-ipa/pull/30
+replace github.com/crate-crypto/go-ipa => github.com/jsign/go-ipa v0.0.0-20221111020207-a50da66c4f8b
```

Comment:
I'm using [optimistically this open PR](https://github.com/crate-crypto/go-ipa/pull/30) with extra improvements on top of what @kevaundray  did to avoid the bytes back and forth. To understand more about this, you can see the Discord discussion.